includes/admin/class-wcmypa-admin.php

```diff
@@ -1190,9 +1192,14 @@ public static function getDeliveryOptionsFromOrder(WC_Order $order, array $input
                 $meta = json_decode(stripslashes($meta), true);
             }
 
+            if ($meta['carrier'] === self::OLD_RED_JE_PAKKETJE_NAME) {
```

Comment:
yoda!

Reply:
En kun je dit in een losse functie zetten? Het is al lelijk genoeg dat het zo moet, maar verstop het dan tenminste in een `normalize...` functie oid